vdr/api/v1/api.go

```diff
@@ -119,6 +119,11 @@ func (a Wrapper) CreateDID(ctx echo.Context) error {
 		}
 	}
 
+	// Enable `KeyAgreement` by default only when it's a DID document owned by the node.
+	if req.KeyAgreement == nil && (req.Controllers == nil || len(*req.Controllers) == 0) {
```

Comment:
not true, the DID Document can be owned by an offline key. I don't see any objections in being it enabled by default for all DID Documents created

Reply:
All right. Then let's do that